temoa/extensions/breakeven/breakeven.py

```
@@ -106,15 +106,15 @@ def coef_FC(instance, target_tech, target_year):
     P_e = instance.time_future.last()
     GDR = value(instance.GlobalDiscountRate)
     MPL = instance.ModelProcessLife
-    LLN = instance.LoanLifetimeProcess
+    LLN = instance.loan_lifetime_process
```

Ruff (0.14.3)

Local variable `LLN` is assigned to but never used

Remove assignment to unused variable `LLN`

(F841)